tmt/steps/report/reportportal.py

```diff
@@ -14,85 +14,124 @@
+             "Appends suite/test description with upload-to-launch/suite options.")
+
     launch_per_plan: bool = field(
         option="--launch-per-plan",
-        default=False,
+        default=_flag_env_to_default('launch_per_plan', False),
         is_flag=True,
         help="Mapping launch per plan, creating one or more launches with no suite structure.")
+
     suite_per_plan: bool = field(
         option="--suite-per-plan",
-        default=False,
+        default=_flag_env_to_default('suite_per_plan', False),
         is_flag=True,
         help="Mapping suite per plan, creating one launch and continous uploading suites into it. "
              "Recommended to use with '--launch' and '--launch-description' options."
              "Can be used with '--upload-to-launch' option to avoid creating a new launch.")
+
     upload_to_launch: Optional[str] = field(
         option="--upload-to-launch",
         metavar="LAUNCH_ID",
-        default=None,
+        default=_str_env_to_default('upload_to_launch', None),
         help="Pass the launch ID for an additional test/suite upload to an existing launch. "
              "ID can be found in the launch URL.")
+
     upload_to_suite: Optional[str] = field(
         option="--upload-to-suite",
         metavar="LAUNCH_SUITE",
-        default=None,
+        default=_str_env_to_default('upload_to_suite', None),
         help="Pass the suite ID for an additional test upload to a suite "
              "within an existing launch. ID can be found in the suite URL.")
+
     launch_rerun: bool = field(
         option="--launch-rerun",
-        default=False,
+        default=_flag_env_to_default('launch_rerun', False),
         is_flag=True,
         help="Rerun the last launch based on its name and unique test paths to create Retry item"
              "with a new version per each test. Supported in 'suite-per-plan' structure only.")
+
     defect_type: Optional[str] = field(
         option="--defect-type",
         metavar="DEFECT_NAME",
-        default=None,
+        default=_str_env_to_default('defect_type', None),
         help="Pass the defect type to be used for failed test, which is defined in the project"
              " (e.g. 'Idle'). 'To Investigate' is used by default.")
+
     exclude_variables: str = field(
         option="--exclude-variables",
         metavar="PATTERN",
-        default="^TMT_.*",
+        default=str(_str_env_to_default('defect_type', "^TMT_.*")),
         help="Regular expression for excluding environment variables "
              "from reporting to ReportPortal ('^TMT_.*' used by default)."
              "Parameters in ReportPortal get filtered out by the pattern"
              "to prevent overloading and to preserve the history aggregation"
              "for ReportPortal item if tmt id is not provided")
 
+    api_version: str = field(
+        option="--api-version",
+        metavar="PATTERN",
```

Comment:
Pattern? That suggests regular expression. Although it looks more like a copy-paste relict...

Reply:
Thanks, it's copy-paste mistake